### ISSUE TITLE
How to migrate or clone a Pinokio Installation, or just change its folder.

### ISSUE THREAD

**Issue by SmoothPlay** [pinokiocomputer/pinokio#97]
I had to mount some hardrives on Ubuntu 23.10, by effect the home path changed. Now I'm not able to just change it or move to another folder without having to install everything all over again, dependencies are always missing even when moving the entire folder.

I still don't get it and I think there should be more documentation about cloning Pinokio installations ou moving them with lesser effort.

I thought Pinokio was some kind of containerized / portable for easily migration and duplication, but apparently it's not that easy.

**Comment by kryztoval** [comment]
Yeah, I am now in the exact same situation. I can't figure out how to tell pinokio to use the folder it was already installed at. if it fails just once to find the folder, it will literally force you to reinstall.

**Comment by DavidEBell** [comment]
I’ve been there a number of times. I reinstall to a new directory then copy the models over. It goes fairly quickly (then I run a backup so I have a clean version handy).

Hope this helps.
D


_______________________________________
David e Bell - Photographer, Writer, Filmmaker

**Comment by kryztoval** [comment]
I ended up renaming my folder to something else
Setting up the program in the folder i had before
Closing it immediately after it was done
Removing new install folder and renaming my folder back to the  original name
Restarting pinokio

I still think this is counter intuitive for an interface that is meant to make things easier. :(

**Comment by DavidEBell** [comment]
Works very well, thanks.
D


_______________________________________
David e Bell - Photographer, Writer, Filmmaker
